lib/mina/application.rb

```diff
@@ -11,10 +11,12 @@ def name
       'mina'
     end
 
+    # :nocov:
     def run
       Rake.application = self
       super
     end
+    # :nocov:
```

Comment:
I removed this method from simplecov coverage because there's no easy way to test this without invoking a task.